### ISSUE TITLE
Extract Not Working

### ISSUE THREAD

**Issue by jag3773** [Door43/dokuwiki-usfm-import#2]
Plugin setup but not working here: https://jp.door43.org/playground/usfmimporter.  The zip files I tried were https://jp.door43.org/PET2015-05-24.zip and https://jp.door43.org/TSI2015-05-24.zip.

**Comment by jag3773** [comment]
I think the problem is the imported file names.  The files should be all lowercase and should end with the .usfm.txt extension.  Example:

43LUKTSI.SFM -> 43luktsi.usfm.txt

**Comment by yvonnel098** [comment]
Yes, the plugin only changes .txt to .usfm.txt.  All other file extensions will be left alone.  I'll change the code to deal with this.  Is there any other file extension I should also change?

**Comment by jag3773** [comment]
​.sfm too.​

Thank you,
Jesse Griffin
unfoldingWord Chief Technology Officer
